src/renderer/dx/DxRenderer.cpp

```diff
@@ -178,7 +179,15 @@ DxEngine::~DxEngine()
         SwapChainDesc.BufferCount = 2;
         SwapChainDesc.SampleDesc.Count = 1;
         SwapChainDesc.AlphaMode = DXGI_ALPHA_MODE_UNSPECIFIED;
-        SwapChainDesc.Scaling = DXGI_SCALING_NONE;
+
+        if (IsWindows8OrGreater())

```

Comment:
Explain. I don't understand why the switch is necessary.

Reply:
`DXGI_SCALING_NONE` is not a valid value on windows 7

Reply:
Leave that as a comment then, please.